src/api/cloudDb/enterprise/user/user.v6.service.js

```diff
@@ -0,0 +1,34 @@
+angular.module('ovh-api-services').service('OvhApiCloudDBEnterpriseUserV6', ($resource, $cacheFactory) => {
+  const cache = $cacheFactory('OvhApiCloudDBEnterpriseUserV6');
+  const queryCache = $cacheFactory('OvhApiCloudDBEnterpriseUserV6Query');
+  const interceptor = {
+    response(response) {
+      cache.remove(response.config.url);
+      queryCache.removeAll();
+      return response;
+    },
+  };
+
+  const userResource = $resource('/cloudDB/enterprise/cluster/:clusterId/user', {
+    clusterId: '@clusterId',
+  }, {
+    query: { method: 'GET', isArray: true, cache: queryCache },
+    get: { method: 'GET', cache },
```

Comment:
You have a `get` but it doesn't seem that you pass a `userId` as a param

Reply:
For a cluster, there can be only 1 user and hence, no userId required. The get method returns this user. The query api has been removed since it was put in here by mistake.